types/react-measure/react-measure-tests.tsx

```diff
@@ -74,11 +93,17 @@ function testHocComponent<T>(): React.ComponentClass<T> {
     ));
 }
 
-function testHocComponent2<T>(): React.ComponentClass<T> {
-    return withContentRect(['scroll', 'margin'])(({measureRef}) => (
+function testHocComponent2<T>() {
+    return withContentRect(['scroll', 'margin'] as ReadonlyArray<MeasurementType>)(({measureRef}) => (
```

Comment:
`['scroll', 'margin']` should satisfy the type `ReadonlyArray<MeasurementType>`, why do you need to cast it?

https://www.typescriptlang.org/play/#src=type%20MeasurementType%20%3D%20'client'%20%7C%20'offset'%20%7C%20'scroll'%20%7C%20'bounds'%20%7C%20'margin'%3B%0A%0Aconst%20foo%3A%20ReadonlyArray%3CMeasurementType%3E%20%3D%20%5B'client'%2C%20'offset'%5D%3B

Reply:
I was getting an error when I ran the tests for DefinitelyTyped. My editor also didn't complain, though, so I found it a bit confusing.

Reply:
I'll take your word for it so I'm not blocking you, but may follow up as the array should satisfy the type without casting.

Reply:
Thanks :) I'll take another look and ensure if it wasn't particular to my development setup.